README.md

```diff
@@ -19,8 +19,8 @@ If you need help using or installing Node.js, please use the
 
 ## Resources for Newcomers
 
-* [Website](https://nodejs.org/en/)
-* [Contributing to the project](./CONTRIBUTING.md)
+* Website: [nodejs.org](https://nodejs.org/en/)
+* Contributing: [CONTRIBUTING.md](./CONTRIBUTING.md)
```

Comment:
Hmmm, IRC is kinda special though because you *could* use the link but usually you'd just copy that into your client. I'm ok for the website, but not the Contributing link.